src/agent/Cargo.toml

```diff
@@ -71,10 +71,10 @@ openssl = { version = "0.10.38", features = ["vendored"] }
 
 # Image pull/decrypt
 [target.'cfg(target_arch = "s390x")'.dependencies]
-image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "v0.4.0", default-features = false, features = ["kata-cc-s390x"] }
+image-rs = { git = "https://github.com/confidential-containers/image-rs", rev = "e7f57a4", default-features = false, features = ["kata-cc-s390x"] }
```

Comment:
can you use a longer hash (either full or atleast 12 chars) so that we don't have problems with non-unique references later

Reply:
Ok!